cacerts/plan.ps1

```diff
@@ -1,41 +1,23 @@
 $pkg_name="cacerts"
 $pkg_origin="core"
-$pkg_version="_set_from_downloaded_cacerts_file_"
+$pkg_description="\
+The Mozilla CA certificate store in PEM format (around 250KB uncompressed).
+"
+$pkg_upstream_url="https://curl.haxx.se/docs/caextract.html"
+$pkg_version="2019.28.08"
```

Comment:
@danielcbright Sorry this has sat open so long. I was getting ready to approve this change and I noticed that this version is going to cause us issues.  We want to use YYYY.MM.DD format, so that it behaves like semver and sorts correctly. YYYY.DD.MM will end up sorting whatever version was released latest in its respective month to the top.

Reply:
@smacfarlane I'll take care of that today!

Reply:
fixed!